Cargo.toml

```
@@ -1,7 +1,9 @@
 [workspace]
 members = [
     "examples/example-runner",
+	"examples/wgpu-example-runner",
```

```suggestion
    "examples/wgpu-example-runner",
```
nit: you used a tab instead of four spaces like the rest of this file.